### ISSUE TITLE
CPU and Memory read state in the wrong direction

### ISSUE THREAD

**Issue by 7m0E7TRWiyc7eJlETAgDR-rp6HLJf2yvSnJLuQL** [Alexays/Waybar#315]
Unlike battery, which reads from 100 to 0, CPU and Memory ought to read from 0 to 100, but they do not with 0.6.2, and instead read the states in the same way that battery is done.  This means that if I have the following states:

> normal: (white)
> warning: 33 (orange)
> critical: 67 (red)

I actually need to apply the css styling (orange and red in this case) for critical to the normal state, normal to the warning state, and warning to the critical state.  Making the css look like this:

> normal: (red)
> warning: 33 (white)
> critical: 67 (orange)

In other words, the css color assignment should be ascending, not descending.

**Comment by Alexays** [comment]
Fixed with 92967c7c06e994528306c99e78d224203b8d1f88

**Comment by unresolvedsymbol** [comment]
I've just built from the latest commit and am still having issues related to this. If I set a state eg. critical to 80, the state is only active once my usage is **below** 80. The only workaround I have found for this is adding a "normal" state and using the not operator in my `style.css`.

**Comment by unresolvedsymbol** [comment]
I have fixed this by adding yet another parameter to the `getState` method which instructs it to use the greater than or equal to operator for determining the state. Will create a pull request shortly.

**Comment by unresolvedsymbol** [comment]
The same flipped behavior occurs with other modules like `memory`, because of this I will make it the default since `battery` is really the only module you want the states to be equal to or less than. Eventually we could add state suport for custom modules as well at this point.

**Comment by unresolvedsymbol** [comment]
#318

**Comment by dlasky** [comment]
thanks i was wondering what was up, also glad to see someone using that feature already!!
